Added vertica support; To use, set type to "vertica" in config; requires...

qzk opened the pull request:
Added vertica support; To use, set type to "vertica" in config; requires vertica-python: https://pypi.python.org/pypi/vertica-python/0.3.0

Note: /opt/vertica/bin/vsql is the standard location for Vertica

JohnMurray commented:
Will review this Monday morning.

JohnMurray commented:
@qzk - Please update `readme.md`'s description to note that it supports Vertica. Also update the requirements to mention / link the vertica driver used.

qzk commented:
@JohnMurray - Added a sample Vertica template and updated the documentation.

As for the question regarding revision_db_name vs revision_schema_name:
In Vertica, it's a much more involved process to create a database,
to the degree that a database must exist before running any SQL (unlike MySQL where SQL can create a database):
http://my.vertica.com/docs/6.0.x/HTML/index.htm#4634.htm

Though, a database name is still required and those are stored in db_name and revision_db_name.
To get around this, a sub-component of the database called "schema" is used.

Your question did also hit on a bug though:
Looking at the code within the db folder,
the Postgres and Vertica interfaces fail to use the revision_db_* config parameters,
and instead use the regular db_* ones.

I've fixed the bug in the provided vertica interface.
I will submit a separate pull request for the postgres interface.

The only time the bug surfaces is when one stores the revision history table in a separate database for postgres / vertica.
